Src/AutoFixture/Kernel/RecursionGuard.cs

```diff
@@ -173,6 +174,9 @@ public RecursionGuard(ISpecimenBuilder builder, IEqualityComparer comparer)
             if (comparer == null)
                 throw new ArgumentNullException("comparer");
 
+            if (recursionDepth < 1)
+                throw new ArgumentOutOfRangeException("recursionDepth");

```

Comment:
Would it be more helpful to the user if the exception message explains what is the acceptable range?